arviz/utils/xarray_utils.py

```diff
@@ -1,18 +1,21 @@
 from abc import ABC, abstractmethod, abstractstaticmethod
-import re
```

Comment:
Does it make sense to call this xarray_utils now?

Reply:
good point - I think we're close to getting rid of some of the original dataframe functions in `utils.py`. probably there should be a refactor then. Really, these classes to convert objects should probably live in their own module, with the `inference_data` object.